robot/variables/load_params.py

```diff
@@ -0,0 +1,24 @@
+import os
```

Comment:
I think we should not add new files to `robot` directory since there is no "robot" anymore

Reply:
good point. moved  to pytest_tests/resources. before merge need to update PYTHOPATH on jenkins.